tests/baselines/reference/genericMappedTypeAsClause.errors.txt

```diff
@@ -38,8 +38,8 @@ genericMappedTypeAsClause.ts(25,60): error TS2312: An interface can only extend
               ~~
 !!! error TS2322: Type 'boolean' is not assignable to type 'MappedModel<T>'.
         const x5: MappedModel<T> = { a: 'bar', b: 42 };  // Error
-                                     ~
-!!! error TS2353: Object literal may only specify known properties, and 'a' does not exist in type 'MappedModel<T>'.

```

Comment:
this was already confusing, `{ a: string, b: number }` is a valid instantiation of `MappedModel<T>`:
```ts
type Model = {
  a: string;
  b: number;
};

type MappedModel<Suffix extends string> = {
  [K in keyof Model as `${K}${Suffix}`]: Model[K];
};

type Compute<T> = { [K in keyof T]: T[K] } & unknown;
declare function f1<T extends string>(): Compute<MappedModel<T>>;

const result = f1<"">();
//    ^? const result: { a: string; b: number; }
```